Yoast/wordpress-seo
# P1-323 Changes to make the new draft-js-plugins in the monorepo work

## Context


* Needed for: https://github.com/Yoast/javascript/pull/1121
* Weirdly, the dependency in `@yoast/replacement-variable-editor` is hoisted in the monorepo, but does then not appear in `wordpress-seo`. The build can then not resolve the import, and consequently crashes. This PR adds the dependencies to `wordpress-seo`, but ideally simply adding `@yoast/replacement-variable-editor` should do the trick.

## Summary


This PR can be summarized in the following changelog entry:

* Adds a dependency that is needed for the @yoast/replacement-variable-editor package.

## Relevant technical choices:

* See context

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* See instructions here https://github.com/Yoast/javascript/pull/1121


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* Go to a post editor, and mouse click in the Focus keyphrase field in the metabox.
* Now start tabbing to the snippet editor fields (seo title and description).
* Verify that you can navigate in out of those fields by tabbing and shift+tabbing.
* Verify that the replacement variables and the input fields work and look as before in all other regards.

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

* This PR affects the following parts of the plugin, which may require extra testing:
	* Every replacement variable editor:
		* Snippet Editor
		* Social Preview forms
		* Replacement variable editors in Yoast settings.

## UI changes


## Documentation


## Quality assurance


Fixes N/A
